pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -259,9 +259,9 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env *model.En
 		}
 
 		listenerMapKey := fmt.Sprintf("%s:%d", endpoint.Address, endpoint.Port)
-		if _, exists := listenerMap[listenerMapKey]; exists {
+		if old, exists := listenerMap[listenerMapKey]; exists {
 			env.PushStatus.Add(model.ProxyStatusConflictInboundListener, node.ID, node,
-				fmt.Sprintf("Rejected %s for %s", instance.Service.Hostname, listenerMapKey))
+				fmt.Sprintf("Rejected %s, used %s for %s", instance.Service.Hostname, old.Service.Hostname, listenerMapKey))
```

Comment:
I think this is outdated.. @nmittler revamped this logic. We now print all services that conflict. So you can undo the changes to this function

Reply:
oh ignore.. this is for inbound